source/h2_stream.c

```diff
@@ -287,6 +291,7 @@ static void s_stream_cross_thread_work_task(struct aws_channel_task *task, void
     bool reset_called;
     size_t window_update_size;
     uint32_t user_reset_error_code;
+    int reset_aws_error_code;
```

Comment:
it would definitely be less confusing to store the combined `struct aws_h2err reset_error_code`

instead of 2 separate variables